### PR TITLE
fix(frontend): Protect quotes in text variables

### PR DESCRIPTION
**Pull request by bodetc**
* Added <sw360:out/> to string variables that could contain quotes

### CODE REVIEW

**Comment by maierthomas** [comment]
related to sw360/sw360portal#534

File: frontend/sw360-portlet/src/main/webapp/html/components/includes/releases/vulnerabilities.jspf [hunk 1]
```diff
@@ -119,7 +119,7 @@
                     '<div title="<sw360:out value='${vulnerabilityVerificationTooltips.get(vulnerability.externalId).get(releaseId)}'/>"><sw360:DisplayEnum value="${vulnerabilityVerifications.get(vulnerability.externalId).get(releaseId)}"/></div>',
                 </core_rt:otherwise>
                 </core_rt:choose>
-                "5": '<div class="dataTables_cell_nowrap">${vulnerability.action}</div>'
+                "6": '<div class="dataTables_cell_nowrap"<sw360:out value="${vulnerability.action}"/></div>'
```
Comment (maierthomas) [inline frontend/sw360-portlet/src/main/webapp/html/components/includes/releases/vulnerabilities.jspf]: Why you changed this value to 6?
The vulnerability value is after the div close.

File: frontend/sw360-portlet/src/main/webapp/html/projects/includes/projects/vulnerabilities.jspf [hunk 2]
```diff
@@ -100,7 +100,7 @@
             <core_rt:forEach items="${vulnerabilityList}" var="vulnerability">
             result.push({
                 "DT_RowId": "${vulnerability.id}",
-                "0": "${vulnerability.intReleaseName}",
+                "0": "<sw360:out value="${vulnerability.intComponentName}"/>",
```
Comment (maierthomas) [inline frontend/sw360-portlet/src/main/webapp/html/projects/includes/projects/vulnerabilities.jspf]: IntReleaseName changed to intComponentName?

File: frontend/sw360-portlet/src/main/webapp/html/projects/includes/projects/vulnerabilities.jspf [hunk 3]
```diff
@@ -125,7 +125,7 @@
                     </core_rt:otherwise>
                 </core_rt:choose>
                 ,
-                "6": '<div class="dataTables_cell_nowrap">${vulnerability.action}</div>'
+                "6": '<div class="dataTables_cell_nowrap"<sw360:out value="${vulnerability.action}"/></div>'
```
Comment (maierthomas) [inline frontend/sw360-portlet/src/main/webapp/html/projects/includes/projects/vulnerabilities.jspf]: Div closing wrong

File: frontend/sw360-portlet/src/main/webapp/html/components/includes/releases/vulnerabilities.jspf (hunk 1 above)
Comment (bodetc) [reply frontend/sw360-portlet/src/main/webapp/html/components/includes/releases/vulnerabilities.jspf]: fixed

File: frontend/sw360-portlet/src/main/webapp/html/projects/includes/projects/vulnerabilities.jspf (hunk 2 above)
Comment (bodetc) [reply frontend/sw360-portlet/src/main/webapp/html/projects/includes/projects/vulnerabilities.jspf]: Copy/pasting error. Fixed.

File: frontend/sw360-portlet/src/main/webapp/html/projects/includes/projects/vulnerabilities.jspf (hunk 3 above)
Comment (bodetc) [reply frontend/sw360-portlet/src/main/webapp/html/projects/includes/projects/vulnerabilities.jspf]: Fixed.

**Comment by bodetc** [comment]
The requested changes were pushed.

**Comment by adrian-evo** [comment]
Releases with quoted versions are not displayed under component release overview. Errors below:

r3

r1

r2

**Comment by bodetc** [comment]
@adrian-evo: I have fixed this last problem.

For all: There might exist more instance of unprotected variables. If so, please create a new ticket.

**Comment by maierthomas** [comment]
Linked release for components with quotes in the version field is working. see attached screenshot.
linked-releases

**Pull request merged**